lambda/service/utils/utils.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import "regexp"
+
+func ExtractRoute(requestRouteKey string) string {
+	r := regexp.MustCompile(`(?P<method>) (?P<pathKey>.*)`)
+	routeKeyParts := r.FindStringSubmatch(requestRouteKey)
+	return routeKeyParts[r.SubexpIndex("pathKey")]
+}
+
+func ExtractParam(rawPath string) string {
+	r := regexp.MustCompile(`/pennsieve-accounts/(?P<accountType>.*)`)
+	tokenParts := r.FindStringSubmatch(rawPath)
+	return tokenParts[r.SubexpIndex("accountType")]
+}
```

Comment:
Can we use the `events.APIGatewayV2HTTPRequest` `PathParameters` map to get the account type rather that writing our own function here?

It will work as long as the path in the OpenAPI spec you put in pennsieve-go-api looks something like `/pennsieve-accounts/{accountType}`. The the value can be retrieved by `request.PathParameters["accountType"]` where request is an `events.APIGatewayV2HTTPRequest`

Reply:
thanks @jfromme - updated.